frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -216,6 +245,8 @@ export const supportLogic = kea<supportLogicType>([
                     comment: {
                         body: (
                             message +
+                            (message2 ? '\n\n' + message2 : '') +
+                            (message3 ? '\n\n' + message3 : '') +
```

Comment:
stuff like `message2`, `message3` is generally not easy to read / extend and hence best to be avoided